### ISSUE TITLE
Cutting polygon with holes to without

### ISSUE THREAD

**Issue by pythonmobile** [google/s2geometry#233]
Is there a function that takes a polygon with holes as input and outpus polygons without holes? (preferably with a small number of diagonals added)

**Comment by jmr** [comment]
No, because S2 doesn't need that. Is there some format that doesn't understand holes that you need this for?
